app/src/ui/dialog/header.tsx

```diff
@@ -63,7 +63,7 @@ export class DialogHeader extends React.Component<IDialogHeaderProps, {}> {
     // I don't know and we may want to revisit it at some point but for
     // now an anchor will have to do.
     return (
-      <a className="close" onClick={this.onCloseButtonClick}>
+      <a className="close" onClick={this.onCloseButtonClick} name="close" role="button">
```

Comment:
It looks like the `name` attribute isn't a valid attribute:

```
[at-loader] ./app/src/ui/dialog/header.tsx:66:8 
    TS2322: Type '{ children: Element; className: string; onClick: (e: MouseEvent<HTMLAnchorElement>) => void; name: string; role: string; }' is not assignable to type 'DetailedHTMLProps<AnchorHTMLAttributes<HTMLAnchorElement>, HTMLAnchorElement>'.
  Property 'name' does not exist on type 'DetailedHTMLProps<AnchorHTMLAttributes<HTMLAnchorElement>, HTMLAnchorElement>'.
```

Reply:
ah sorry about that! @HirdayGupta it looks like `name="close"` should actually be `aria-label="close"`. my bad! after we make that adjustment, this should be good to go!

Reply:
Done!